javascript-timers/interval.js

```diff
@@ -0,0 +1,19 @@
+var $blastOff = document.querySelector('.countdown-display');
+$blastOff.addEventListener('click', letsGo);
+
+function letsGo() {
+  setInterval(startCountdown, 1000);
+}
+
+function startCountdown() {
+  var $countdown = document.querySelector('.countdown-display');
+  if ($countdown.textContent === '4') {
+    $countdown.textContent = '3';
+  } else if ($countdown.textContent === '3') {
+    $countdown.textContent = '2';
+  } else if ($countdown.textContent === '2') {
+    $countdown.textContent = '1';
+  } else if ($countdown.textContent === '1') {
```

Comment:
This will work, but it is very manual. What if the timer needed to start at 1000? That would be a lot of conditional blocks 😬